inferentia/scripts/setup.sh

```diff
@@ -96,11 +96,13 @@ for OPTS; do
         ;;
         -inf2|--inf2-setup)
         INSTALL_INF2=1
+        INSTALL_INF1=0
```

Comment:
I think for completeness, similar handling should be done if `-inf1` is specified (line 92), i.e. set `INSTALL_INF2=0` even it is 0 by default.

Reply:
Thank you Guan for the quick review, just updated - if all looks good, request you to merge